interface/weno/adminClass.php

```diff
@@ -0,0 +1,723 @@
+<?php
+
+
+
+class adminProperties {
+
+	public function __construct(){}
+
+	/*
+	*   Check to see if the database tables exist
+	*/
+
+	public function dataBaseTableExist(){
+        
+        $dbase = $GLOBALS['dbase'];  //incase there are multiple installs on the same server
+
+        $tables = array("pharmacies_weno","drug_paid","prescription_rx_log","narcotics");
+
+		$sql = "SELECT table_name FROM information_schema.tables WHERE table_name = ? AND table_schema = ? ";
+        
+         $missing = array();
+        foreach($tables as $table){
+            $isMissing = sqlQuery($sql, array($table,$dbase));
+            $missing[] = $isMissing['table_name'] ;
+        }
+
+        return $missing;
+	}
+
+
+
+    public function createTables(){
+
+
+    	sqlStatement(
+              "CREATE TABLE IF NOT EXISTS `drug_paid` (
+                            `drugid` int(11) NOT NULL AUTO_INCREMENT,
+                            `drug_label_name` varchar(45) NOT NULL,
+                            `AHFS_descr` varchar(45) NOT NULL,
+                            `NDC` bigint(12) NOT NULL,
```

Comment:
(This comment is in the interface/weno/adminClass.php script)

Better practice to avoid caps in database labels for the future. Using in column labels are ok but frowned upon. table names will break(which you are not doing).